itou/templates/signup/siae_select.html

```diff
@@ -139,12 +139,10 @@ <h3 class="h2">Entreprises déjà inscrites</h3>
                             {% if siae.address_line_2 %}{{ siae.address_line_2 }},{% endif %}
                             {{ siae.post_code }} {{ siae.city }}
                             <br>
-                            {% if siae.active_admin_members %}
-                                {% with siae.active_admin_members.first as admin %}
-                                    {# For security, display only the first char of the last name. #}
-                                    <i>Pour obtenir une invitation, contactez {{ admin.first_name|title }} {{ admin.last_name|slice:1 }}.</i>
-                                {% endwith %}
-                            {% endif %}
+                            {% with siae.memberships.all.0.user as admin %}
```

Comment:
Je comprends que tu remplaces l'affichage du premier user avec un rôle d'admin dans la Siae, par le premier user de la Siae. Cohérent avec le commentaire ` # A user cannot join structures that already have members.`

Reply:
hmmm en effet c'est un peu une erreur ça :/ il faudrait que je sélectionne le premier "admin"  et là ce n'est pas garanti, quel beta. Je vais vérifier que ça fonctionne quand meme ou corriger

Reply:
corrigé.